tests/unit/IonTextWriterTest.js

```diff
@@ -367,41 +375,61 @@
     // PrettyPrint
     skippedPrettyTest('Writes composite pretty ion',
       writer => {
-        writer.stepIn(ion.IonTypes.STRUCT, ['a1']);
+        writer.setAnnotations(['a1']);
```

Comment:
Food for thought: maybe in a future release we could allow annotations to be specified either as an array or as [varargs-style strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters).
```typescript
writer.setAnnotations(['a1']);
writer.setAnnotations(['a1', 'a2', 'a3']);
//   vs.
writer.setAnnotations('a1');
writer.setAnnotations('a1', 'a2', 'a3');
```

Reply:
opened #372 for you